cognitedata/reveal
# feat(react-components): add load error handling capabilities

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/UX-1349

## Description :pencil:


Add load error handling:
- Props for providing error handlers in both model containers and `Reveal3DResources`. 
- Default error handling using `console.warn`.

## How has this been tested? :mag:




## Test instructions :information_source:




In the Reveal3DResources story, selectively make each resource invalid by modifying IDs. Observe that the (`Reveal3DResources`) error handler is run and the console is filled with warnings.


## Checklist :ballot_box_with_check:
